### PR TITLE
Block translation

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Resources/translations/SonataMediaBundle.fr.xliff [hunk 1]
```diff
@@ -441,6 +441,18 @@
                 <source>warning_no_category</source>
                 <target>Pas de catégories définies</target>
             </trans-unit>
+            <trans-unit id="sonata.media.block.media">
+                <source>sonata.media.block.media</source>
+                <target>sonata.media.block.media</target>
```
Comment (OskarStark) [inline Resources/translations/SonataMediaBundle.fr.xliff]: ping @Soullivaneuh @rande can you help with the French translation?

Comment (rande) [inline Resources/translations/SonataMediaBundle.fr.xliff]: sonata.media.block.media => Medium
sonata.media.block.feature_media => Media + Texte
sonata.media.block.gallery => Gallerie

Comment (OskarStark) [inline Resources/translations/SonataMediaBundle.fr.xliff]: Thank you @rande

**Comment by core23** [comment]
Thanks @rande

**Comment by OskarStark** [comment]
ping @sagikazarmark

**Comment by sagikazarmark** [comment]
``` xml
<trans-unit id="sonata.media.block.media">
    <source>sonata.media.block.media</source>
    <target>Média</target>
</trans-unit>
<trans-unit id="sonata.media.block.feature_media">
    <source>sonata.media.block.feature_media</source>
    <target>Kiemelt médie</target>
</trans-unit>
<trans-unit id="sonata.media.block.gallery">
    <source>sonata.media.block.gallery</source>
    <target>Galéria</target>
</trans-unit>
```

hu

**Comment by core23** [comment]
Thanks @sagikazarmark 

Added to the right file ;) @OskarStark

**Comment by OskarStark** [comment]
:smile: :+1:

File: Resources/translations/SonataMediaBundle.de.xliff [hunk 2]
```diff
@@ -440,7 +440,19 @@
             <trans-unit id="form.label_provider_reference">
                 <source>form.label_provider_reference</source>
                 <target>Provider Referenz</target>
-            </trans-unit>            
+            </trans-unit>
+            <trans-unit id="sonata.media.block.media">
+                <source>sonata.media.block.media</source>
+                <target>Medium</target>
```
Comment (OskarStark) [inline Resources/translations/SonataMediaBundle.de.xliff]: "Medium" is right? sounds weird?

"Media" is better IMO, what do you think @core23

Comment (rande) [inline Resources/translations/SonataMediaBundle.de.xliff]: Medium is the correct word: http://www.merriam-webster.com/dictionary/medium. Media is the plurial of medium. Sound weird ;)

Comment (core23) [inline Resources/translations/SonataMediaBundle.de.xliff]: As @rande said. The German singular is for *Medien* is *Medium* 
http://www.duden.de/rechtschreibung/Medium_Vermittler_Traeger

Comment (OskarStark) [inline Resources/translations/SonataMediaBundle.de.xliff]: :+1:

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**